src/styles/blocks/cards.css

```diff
@@ -0,0 +1,74 @@
+.width__card {
```

Comment:
It is something strange. Use understandable names and move it to a separate file. One block per file. Better call body `.page` and do `.page__card` for this.